src/hooks/useScanResults.ts

```diff
@@ -105,12 +116,13 @@ export const useScanResults = (
 };
 
 export const getScanResultsMap = (
-  taskRuns: TaskRunKind[],
-): { [key: string]: [ScanResults, TaskRunKind[]] } => {
+  taskRuns: (TaskRunKind | TaskRunKindV1Beta1)[],
```

Comment:
```suggestion
  taskRuns: (TaskRunKind)[],
```
```
type TaskRunKind = TaskRunKindV1 | TaskRunKindV1Beta1
```
It will be easy to remove the deprecated type once v1 is fully supported

Reply:
Thanks, updated.